content/blog/2020/11/2020-11-03-gsoc-report.adoc

```diff
@@ -0,0 +1,163 @@
+Please refer to the Jenkins online meetup recordings for the recent versions.
+
+GSoC Mentor summit::
+This is a regular gathering for Google Summer of Code mentors and org admins where they share their experiences about GSoC, outreach programs, community management and tools.
+Usually it is organized as a multi-day unconference after the end of GSoC, with 2-3 representatives from each project.
+It has been a great learning experience to participate in it.
+This year it was a single-day virtual event, and all mentors were able to attend.
+link:/blog/authors/shivaylamba/[Shivay Lamba], one of the GSoC 2020 mentors, also did a lightning talk about the GSoC projects he was working on in Jenkins and CNCF (link:https://drive.google.com/file/d/1Ian5JvKp9UuAg3k7NnGPefgFwIxQfpJm/view[slides]).
+
+== Swag
+
+All Google Summer of Code students and mentors get swag from Google.
+This year, link:https://cd.foundation/[Contrinuos Delivery Foundation (CDF)] has sponsored swag for 50 most active GSoC participants:
+all students, mentors, and many other contributors who participated and helped the projects to succeed.
+This is the third year when the Jenkins organization sends extra GSoC swag,
+In the previous years the swag logistics was one of the most challenging tasks for org admins during the entire project.
+and we highly appreciate help from CDF with this part.
+As DevOps World presenters, the students have also received special edition speaker swag from CloudBees.
+
+image:/images/gsoc/2020/summary_report/swag.jpg[GSoC swag by the Jenkins project]
+
+Thanks a lot to link:https://opensource.google/[Google], link:https://cd.foundation/[Contrinuos Delivery Foundation (CDF)] and link:https://www.cloudbees.com/[CloudBees]!
+
+== Retrospective
+
+After completion of the coding phases,
+org admins have reached out to all GSoC 2020 participants to gather their feedback and suggestions.
+We also recommended that project teams hold their own retrospective meetings.
+Such information is instrumental to continuously improving GSoC in the Jenkins community.
+We thank all contributors who shared their feedback!
+
+The organization-wide retrospective was organized as a survey and a series of retrospective meetings.
+You can find aggregated results in link:https://docs.google.com/document/d/1NIszUtuXmHiu8X2WrgAEQFK6aVodsmM4I0RSNRf4TS0/edit?usp=sharing[this Google Doc].
+Overall, we have got a very positive feedback from students and mentors.
+The GSoC framework in Jenkins has matured a lot during the previous years,
+and we have invested a lot in creating guidelines and recommendations for all parties.
+This year these guidelines helped a lot because all the expectations were known in advance.
+As usual, there is a lot to improve, especially with regards to the community bonding phases and cross-project communications.
+We are processing the feedback, and we will expand our documentation and the contributor onboarding plans next year.
```

Comment:
Minor phrasing changes (optional)
```suggestion
The GSoC framework in Jenkins has matured significantly during the previous years.
The effort we invested to create guidelines and recommendations for all parties helped a lot because all the expectations were known in advance.
As usual, there is much to improve, especially with regards to the community bonding phases and cross-project communications.
We are processing the feedback, and we will expand our documentation and the contributor onboarding plans next year.
```